lepton/openstack.go

```diff
@@ -390,10 +392,16 @@ func (o *OpenStack) GetInstances(ctx *Context) ([]CloudInstance, error) {
 		for _, s := range serverList {
 			// fugly
 			ipv4 := ""
+<<<<<<< HEAD
```

Comment:
Some conflicts here, can you please rebase your PR?

Reply:
thanks for the heads up @sanketsudake